viewer.py

```diff
@@ -315,6 +317,12 @@ def setup():
 
     signal(SIGUSR1, sigusr1)
     signal(SIGUSR2, sigusr2)
+    
+    c = ntplib.NTPClient() 
+    responce = c.request('europe.pool.ntp.org', version=3) 
+    cur_date = ctime(responce.tx_time) 
+    sudodate = subprocess.Popen(["sudo", "date", "-s", cur_date]) 
```

Comment:
This won't work unfortunately because we cannot assume we can run `sudo` without a password.